packages/lab/src/tabs/Tab.css

```diff
@@ -83,42 +84,39 @@
   display: inline-block;
   position: relative;
   overflow: hidden;
-  text-align: center;
+  text-align: var(--uitk-text-textAlign-embedded);
   text-overflow: ellipsis;
-  top: var(--uitkTab-top, auto);
+  top: var(--uitkTabs-tab-top, var(--tabs-tab-top, auto));
   white-space: nowrap;
-  /* ensure content sita above focus ring */
+  /* ensure content sits above focus ring */
   z-index: var(--uitk-zIndex-default);
 }
 
 .uitkTab-vertical .uitkTab-text {
-  text-align: left;
+  text-align: var(--uitk-text-textAlign);
 }
 
 .uitkTab .uitkTab-text:before {
   height: 0;
   content: attr(data-text);
   display: block;
   visibility: hidden;
-  /* must match font weight of selected tab styling */
-  font-weight: var(--uitk-typography-weight-semiBold);
+  font-weight: var(--uitk-navigable-fontWeight-active);
 }
 
-/* Editing  - these are focusVisible styles, just assign uitkFocusVisible*/
+/* FIXME: these are all focusVisible styles, but with a bespoke inset */
 .uitkTab-editing:after {
   content: "";
   position: absolute;
   top: 0;
   left: 0;
   right: 0;
   bottom: 2px;
-  /* border: dotted cornflowerblue 2px;
-   */
+  inset: 2px 2px 4px 2px;
```

Comment:
inset sets top bottom right and left, don't need those individual values as well